README.md

```diff
@@ -113,6 +113,27 @@ Assuming an existing HomeBrew installation:
 % brew install boost cmake gcc libpcap pkg-config ragel sqlite
 ```
 
+### *BSD
+In NetBSD you will almost certainly need to have a newer compiler installed. 
+This will not replace the one in the standard base distribution, and
+cmake will probably find the wrong compiler when it checks automatically.
+Using the example of gcc12 from pkgsrc, one will need to set two
+environment variables before starting: 
+```
+export CC="/usr/pkg/gcc12/bin/cc"
+export CXX="/usr/pkg/gcc12/bin/g++"
+```
+
+In FreeBSD similarly, you might want to install a different compiler.
+Using the example of gcc12 from pkg:
+```
+export CC="/usr/local/bin/gcc"
+export CXX="/usr/local/bin/g++"
+```
+
```

Comment:
Could you also add instructions on how to install the required dependencies on FreeBSD?